posthog/warehouse/models/datawarehouse_saved_query.py

```diff
@@ -93,6 +100,10 @@ class Status(models.TextChoices):
         related_name="saved_queries",
     )
 
+    source = models.CharField(
```

Comment:
`source` is an overloaded term in data warehouse. Can we call it `origin` instead? I know it's the same thing, but it removes ambiguity that it's related to the `ExternalDataSource` model